pkg/autoscaler/autoscaler/util.go

```diff
@@ -33,6 +34,10 @@ const (
 
 	invalidMemberTypeErrorMsg    = "tac[%s/%s] invalid set MemberType:%s"
 	invalidTacAnnotationErrorMsg = "tac[%s/%s] annotation invalid set,err:%v"
+
+	// lastSyncThresholdSec is maximum time allowed before reset phase status
+	// 30 is the resync during, 3 is a factor
+	lastSyncThresholdSec = 30 * 3
```

Comment:
Replace 30 as `controller.ResyncDuration`, so the lastSyncThreshold would be 3 times ResyncDuration